admin/app/components/certifications/certification/informations/global-actions.gjs

```diff
@@ -37,15 +38,19 @@ export default class CertificationInformationGlobalActions extends Component {
   }
 
   get displayRejectCertificationButton() {
-    return this.args.certification.status !== 'rejected';
+    return this.args.certification.status === 'validated';
```

Comment:
Si les règles sont les mêmes pour les 3 boutons alors je propose de faire passer par une seule et même fonction, ne serait-ce que pour le côté DRY, mais aussi pour la cohérence.
Ici pourquoi alors le bouton de rejet n'a pas les arguments `!this.args.certification.isPublished && this.args.session.finalizedAt` comme les autres ?

Reply:
Autre point : ceci bloque côté frontend ici.
Mais on ne peut pas faire confiance aux utilisateurs, as-tu pu le vérifier lors de ton implém' que le backend bloque ? est-ce que le backend bloquait bien (erreur 500 ou autre) si on envoie une requete au backend et que la session est finalisée par exemple ?

je pose la question car je crois me souvenir que l'origine est que le statut est chelou quand on cliquait sur le bouton, du coup cela m'évoquait une faille côté backend (en plus que l'interface permette de faire l'action)

Reply:
PS : entre temps ou j'ai ouvert et maintenant je viens de voir que tu as retiré les labels de review 😅 mais mon github etait sur la relecture et avait pas refresh haha

<img width="912" height="97" alt="image" src="https://github.com/user-attachments/assets/99f5c906-dd00-4486-b8e7-ec2b49cc58c9" />